### ISSUE TITLE
Codemirror vim ":" issue

### ISSUE THREAD

**Issue by TonyZTYang** [laurent22/joplin#3603]
When using vim mode in codemirror editor, the vim command ***:*** does not trigger an input field.

## Environment

Joplin version: Joplin 1.0.233 (prod, darwin)
Platform: macOS
OS specifics: macOS catalina 10.15.5 (19F101)

## Steps to reproduce

1. Open Joplin
2. Set keyboard mode to vim, set editor to codemirror
3. Open any note in editor view and focus on the note body (in vim command mode)
4. type ***:*** symbol

## Describe what you expected to happen
An input box should appear enabling input of functions such as `:%s/old/new/g` just as it is in default ace editor.
It seems that the keystrokes after ":" are logged in and can be executed. But the input box along with inputed text is not visible.

## Logfile

No extra logs were generated when recreating this issue but there is a long list of warnings in console when loading the app.
[warning.txt](https://github.com/laurent22/joplin/files/5034198/warning.txt)

**Comment by TonyZTYang** [comment]
update: it seems that the bug also occurs when searching in vim command mode using _**/**_ - no input field appears but the search can be executed.

**Comment by laurent22** [comment]
@CalebJohn, the input field is indeed there but it's pushed down by some padding in the text area above. Could it be related to the scroll past end feature? Or maybe the height of the entire component is somehow incorrect?

In the screenshots below, CodeMirror-dialog is the input field and by removing the padding from CodeMirror-scroll I can make it appear, but I suspect it wouldn't be the right fix.

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/1285584/89525109-9f628680-d7dd-11ea-891b-4cc955d0adc9.png">

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/1285584/89525143-ac7f7580-d7dd-11ea-9e4c-cf889420ad5a.png">

**Comment by CalebJohn** [comment]
@laurent thanks for tagging me in this. I did a bit of testing and figured out the cause, which is that I had forgotten to add the dialog box css. While working on the initial Codemirror editor I had included this css while testing in editor search, I removed it after deciding to go with Joplins existing search facility. I would have removed the css file at that time forgetting to check if it broke anything else.

I'll go ahead and prepare a fix for this by adding in proper css loading (copying tinymce).
